integration/experiment/launch_util.py

```diff
@@ -168,6 +168,9 @@ def launch_all_runs(targets, num_nodes, iterations, extra_cli_args, output_dir,
                     # Hit if e.g. the app calls MPI_ABORT
                     sys.stderr.write('Warning: <geopm> Execption encountered during run {}'.format(e))
                     sys.stderr.write('Retrying previous trial...')
+                    # Without sleep, the failed run tries to start over and over again and becomes
```

Comment:
Why is the run failing?

Reply:
It is not because of iterations but retries independent of iteration counts. It seems that the purpose here was if an experiment failed it will be retried. We might also want to put a limit on how many times it is retried.

Reply:
Because of this: https://github.com/geopm/geopm/blob/d0e4506c924832b946f20c8c376e9c0457c6af05/integration/experiment/launch_util.py#L159

Even with 1 iteration, it never reaches the completed state. I think we should also put a limit on this.

Reply:
Ah, I see.  Thanks for the explanation.
Yeah, I wrote this for when GADGET does a bad walk of the simulation and aborts.  In that case, I wanted to keep retrying until it ran successfully.
I agree that the number of attempted retries should be capped (~10 or something).
This is fine for now.